packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page.tsx

```diff
@@ -29,6 +32,17 @@ import {
 } from "../../../shared/layout";
 import { BlockBasedEntityEditor } from "./[entity-id].page/block-based-entity-editor";
 import { useRouteAccountInfo } from "../../../shared/routing";
+import { TopContextBar } from "../../shared/top-context-bar";
+
+const Container = styled("div")(({ theme }) => ({
+  display: "grid",
+  gridTemplateColumns: "1fr minmax(65ch, 1200px) 1fr",
+  padding: theme.spacing(7, 10),
+
+  "& > *": {
+    gridColumn: "2",
+  },
+}));
```

Comment:
I don't feel there's a need to create a reusable component since different pages have different requirements. For example pages (as in page entities) have a different max-width compared to entity or entity type pages.